server/src/main/java/io/druid/initialization/Initialization.java

```diff
@@ -132,9 +111,7 @@ static void clearLoadedModules()
   }
 
   /**
-   * Look for extension modules for the given class from both classpath and druid.extensions.coordinates.
-   * Extensions explicitly specified in druid.extensions.coordinates will be loaded first, if there is a duplicate
-   * extension from classpath, it will be ignored.
+   * Look for extension modules for the given class from both classpath and extensions directory.
```

Comment:
can we clarify which one takes precedence? will it prefer the extension found on the classpath or the one from the directory?